kernel/traffic/core/src/main/java/org/apache/shardingsphere/traffic/algorithm/traffic/segment/SQLMatchTrafficAlgorithm.java

```diff
@@ -43,6 +46,8 @@ public final class SQLMatchTrafficAlgorithm implements SegmentTrafficAlgorithm {
     public void init(final Properties props) {
         Preconditions.checkArgument(props.containsKey(SQL_PROPS_KEY), "%s cannot be null.", SQL_PROPS_KEY);
```

Comment:
Please use `ShardingSpherePreconditions` to replace `Preconditions`, Thank you.

Reply:
It's wired, I have made the modification in the last PR.
I make the review as your advice again.

Reply:
Please check if the code was pushed successfully, I see no modification.

Reply:
Please check if the code was pushed successfully, I see no modification.